controllers/getOneArticleWithComments.controller.js

```diff
@@ -0,0 +1,14 @@
+const fetchOneArticleWithComments = require("../models/fetchOneArticleWithComments.model.js");
+
+function getOneArticleWithComments(req, res, next) {
+  const { article_id } = req.params;
+  fetchOneArticleWithComments(article_id)
+    .then((comments) => {
+      res.status(200).send({ comments: comments });
```

Comment:
You can simply send {comments}, this is a shorthand for {comments: comments} JS will take care of it, again just saves on code